### ISSUE TITLE
Improve the CSS intellisense

### ISSUE THREAD

**Issue by abdonrd** [mjbvz/vscode-lit-html#43]
The CSS intellisense autocompletes with HTML:

![jul-10-2018 14-45-59](https://user-images.githubusercontent.com/1007051/42510694-0053c758-8450-11e8-9be8-f1af5ac206de.gif)

**Comment by justinribeiro** [comment]
So, this is a bit of an odd one. I can reproduce this only in two cases at the moment: 

1. If you recently update the plugin and don't restart code, this occurs. 
2. If you click the reload button after updating this plugin, this occurs.

My suspicion is because the new language service for which the plugin depends on doesn't load. If you restart code, do you get the same behavior?

**Comment by abdonrd** [comment]
@justinribeiro yes... now I have closed the VSCode. Then open only this `my-element.js` file and:

![jul-10-2018 19-12-35](https://user-images.githubusercontent.com/1007051/42526158-5ae621a0-8475-11e8-90ec-3f519a87f8d3.gif)

This time I have written slower to be able to see well the intellisense.

**Comment by abdonrd** [comment]
@mjbvz if we remove it from `emmet.includedLanguages`, we will not have autocomplete support in HTML or CSS. 😕 

Reference: https://github.com/mjbvz/vscode-lit-html/issues/32#issuecomment-401901487

**Comment by justinribeiro** [comment]
I have an older branch sitting that explicitly adds (well, attempts to smooth) the emmet support. Let me rebase it and see. Emmet as a whole is a bit of an uneven experience in my testing, as there are some issues open upstream.

**Comment by davie-robertson** [comment]
I think the setting has now changed to:
` emmet.excludeLanguages": [ ],`

**Comment by justinribeiro** [comment]
I _had_ a working version of this but I ran short on time in the summer. With the holidays fast approaching, I'm planning on circling back since I'll have some time freed up (all things equal).

**Comment by abdonrd** [comment]
@justinribeiro great, thanks Justin!

**Comment by abdonrd** [comment]
@mjbvz could you reopen the issue? Thanks in advance!

**Comment by abdonrd** [comment]
BTW, is this related?

![nov-17-2018 01-07-09](https://user-images.githubusercontent.com/1007051/48653406-3a616e80-ea05-11e8-9f68-111b15240159.gif)

The host selector tooltip is different. Also the tooltip title color is different (blue vs green).

**Comment by abdonrd** [comment]
Friendly ping to @mjbvz 🙂

**Comment by alejost848** [comment]
@justinribeiro @abdonrd Was this ever resolved? I'm just starting with `lit-element` and I'm facing the same issue with autocomplete. Right now either only HTML or CSS work at a time, but not both.

`
"emmet.includeLanguages": {
        "javascript": "html"
    }`
or
`
"emmet.includeLanguages": {
        "javascript": "css"
    }`

**Comment by abdonrd** [comment]
@alejost848 now I'm using the [lit-plugin](https://github.com/runem/lit-analyzer/tree/master/packages/vscode-lit-plugin).
